### PR TITLE
[fix](memory) Allows to enable memory tracker accuracy detection via Config

### PR DESCRIPTION
**Pull request by xinyiZzz**
add `crash_in_memory_tracker_inaccurate`, If memory tracker value is inaccurate, BE will crash. usually used in test environments, default value is false.

### CODE REVIEW

**Comment by xinyiZzz** [comment]
run buildall

File: be/src/common/config.cpp [hunk 1]
```diff
@@ -150,6 +150,9 @@ DEFINE_mInt64(stacktrace_in_alloc_large_memory_bytes, "2147483648");
 
 DEFINE_mInt64(crash_in_alloc_large_memory_bytes, "-1");
 
+// If memory tracker value is inaccurate, BE will crash. usually used in test environments, default value is false.
+DEFINE_mBool(crash_in_memory_tracker_inaccurate, "false");
```
Comment (yiguolei) [inline be/src/common/config.cpp:154]: also modify this parameter in p0 p1 external table pipeline

**Review by yiguolei — DISMISSED** [review]

File: be/src/common/config.cpp (hunk 1 above)
Comment (xinyiZzz) [reply be/src/common/config.cpp:154]: done

**Comment by xinyiZzz** [comment]
run buildall

**Comment by xinyiZzz** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]

**Review by wangbo — APPROVED** [review]
LGTM

**Pull request merged**